datalab/bigquery/_query.py

```diff
@@ -332,14 +345,17 @@ def sample(self, count=5, fields=None, sampling=None, use_cache=True):
       fields: the list of fields to sample (default None implies all).
       sampling: an optional sampling strategy to apply to the table.
       use_cache: whether to use cached results or not (default True).
+      use_legacy_sql: if True, query will run using BigQuery's legacy SQL dialect. If False,
+          query will run using BigQuery's standard SQL (beta).
     Returns:
       A QueryResultsTable containing a sampling of the result set.
     Raises:
       Exception if the query could not be executed or query response was malformed.
     """
     return Query.sampling_query(self._sql, self._context, count=count, fields=fields,
                                 sampling=sampling, udfs=self._udfs,
-                                data_sources=self._data_sources).results(use_cache=use_cache)
+                                data_sources=self._data_sources).results(use_cache=use_cache,
+                                                                         use_legacy_sql=use_legacy_sql)
 
   def execute_dry_run(self):
```

Comment:
I think we should also add the same keyword argument ('use_legacy_sql=True') here.